Meta/build-image-qemu.sh

```diff
@@ -50,15 +37,6 @@ if [ "$(uname -s)" = "Darwin" ]; then
 
     E2FSCK="e2fsck"
     RESIZE2FS_PATH="resize2fs"
```

Comment:
Can't select the whole block here because it's outside the diff, but would it make sense to move the e2fsck and resize2fs detection logic for Darwin to `.shell_include.sh` as well?

Reply:
I didn't want to touch the Darwin stuff as I can't test it, but I guess that we can trust the CI.